src/components/atoms/IconComponent/Icon.scss

```diff
@@ -12,5 +12,17 @@
     &__large {
```

Comment:
After adjustments, can you add a test modal on feed screen, just for testing purposes?

Reply:
Sure - I hope I did it the right way